### PR TITLE
Camel 2.21.x

### PR DESCRIPTION
**Pull request by tdiesler**

### CODE REVIEW

File: feature/modules/src/main/resources/modules/system/layers/fuse/org/apache/qpid/proton/0.25/module.xml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<module xmlns="urn:jboss:module:1.1" name="org.jgroups" slot="4.0">
+<module xmlns="urn:jboss:module:1.1" name="org.apache.qpid.proton" slot="0.25">
```
Comment (jamesnetherton) [inline feature/modules/src/main/resources/modules/system/layers/fuse/org/apache/qpid/proton/0.25/module.xml]: Minor nitpick - slot should be 0.26 to reflect the dependency upgrade.

**Comment by tdiesler** [comment]
retest this please

**Comment by tdiesler** [comment]
retest this please

**Comment by tdiesler** [comment]
retest this please

**Pull request merged**